### ISSUE TITLE
Error when a new page created with title name in Chinese

### ISSUE THREAD

**Issue by yanghaizhou1030** [pluck-cms/pluck#27]
It seems that pluck can not display a page with tile name in Chinese

**Comment by billcreswell** [comment]
Can you tell us anything more about what happens?

**Comment by noodlefighter** [comment]
@billcreswell 
New page named "测试", save.
```
Warning: require_once(/home/soft/public_html/Pluck/data/settings/pages) [function.require-once]: failed to open stream: Invalid or incomplete multibyte or wide character in /home/soft/public_html/Pluck/data/inc/editpage.php on line 22

Fatal error: require_once() [function.require]: Failed opening required 'data/settings/pages/' (include_path='.:/usr/local/apps/php53/lib/php') in /home/soft/public_html/Pluck/data/inc/editpage.php on line 22
```

If you try to access the page(like"http://xxx.com/Pluck/?file=测试"), it says:
```
404: not found
This page could not be found.
```

**Comment by BSteelooper** [comment]
I think that this happens when the friendly url cannot be translated. There is no Chinese to English converter. Now it enters double byte characters as the friendly name, which is also the filename. 

We should ask for a valid friendly url if the characters are not latin based?

I found this piece of code which can detect is a double byte character is used.
`var regex = /[^\u0000-\u00ff]/; // Small performance gain from pre-compiling the regex
function containsDoubleByte(str) {
    if (!str.length) return false;
    if (str.charCodeAt(0) > 255) return true;
    return regex.test(str);
}`

but what to do next?
* Return an error and ask for the friendly URL?
* Generate a friendly url and save anyway?

**Comment by BSteelooper** [comment]
fix for issue #27 if non latin text is used as the title, the filename is replaced with the date to prevent breaking Pluck.
The SEO Name is no longer SEO and not a representation of the Title, but translating UNICODE to Latin in pluck is not doable
